### ISSUE TITLE
Angular.js Field Validation does not work

### ISSUE THREAD

**Issue by IvandaNothabeer** [Fijo/EVE-Composition-Planer#13]
If an invalid entry is made to any field, the field border color turns red, but no popup error message appears and the field cannot be edited any more. The record cannot be saved.

I'm somewhat worried that this is due to a revision mismatch in the angular libraries. I notice that my build is at a later revision level that of the live site. However, forcing my build to the latest release of all angular libraries does not resolve the issue, but instead breaks some of the angular-animate libraries even worse.

**Comment by Fijo** [comment]
ok so for the libraries that dont have the version as specified in the bower.json could you please tell me which version you are running instead for each of those libraries so i can do some testing for myself.

If you can I'd recommend just going with the exact same libs as in that bower.json file. I might go at it and update them but for know I know that this version is well somewhat stable ish.

I'm gona have a closer look at this tomorow since I'm quite tired right now.

**Comment by Fijo** [comment]
good news!
I managed to reproduce the issue with the current bower installer routine tonight. I corrected some of the versions according to what's installed on the server and made the versions more fix to insure the same functioning when being used on differed installations.
That does not mean I won't be updating them someday but for now those versions are known to work best.

to update in the ./public directory of your repo do this:
rm -r bower_components
nodejs ../node_modules/bower/bin/bower install

Don't forget to redeploy the new version to your apache2 server public directory afterwards.
Also you might have to do a hard refresh (control + f5) on the webpage to get the new versions of the libraries.

Thanks for feedback!
